hc/api/management/commands/sendalerts.py

```diff
@@ -68,3 +69,39 @@ def handle(self, *args, **options):
             if ticks % 60 == 0:
                 formatted = timezone.now().isoformat()
                 self.stdout.write("-- MARK %s --" % formatted)
+                self.pinged_often()
+
+
+    def pinged_often(self):
+        """
+        -get the check that was last pinged
+        -check the time lag between last ping and the previous ping
+        -if it's greater than the reverse grace, then PINGED QUITE OFTEN
+        -equalise the "previous_ping" and "last_ping" to avoid repeating the PINGED QUITE OFTEN message
+        """
+        try:
```

Comment:
What's the purpose of the try block if the except part is simply returning?

Reply:
The `try block` is to avoid errors because when a check has not yet pinged, its value of `previous_ping` in the database is `null` and the subtraction of `last_ping` and `previous_ping` throws an exception which I want to catch and simply return, since at this point ,clearly the ping hasn't been pinged many times than it should.